pkg/webhooks/sgx/sgx.go

```diff
@@ -104,6 +106,16 @@ func (s *SgxMutator) Handle(ctx context.Context, req admission.Request) admissio
 			return admission.Errored(http.StatusInternalServerError, err)
 		}
 
+		_, ok := requestedResources[encl]
+		if ok {
+			warnings = append(warnings, encl+" should not be used in Pod spec directly")
+		}
+
+		_, ok = requestedResources[provision]
+		if ok {
+			warnings = append(warnings, provision+" should not be used in Pod spec directly")
+		}
```

Comment:
Looks good. This can be moved to a function returning updated `warnings` slice:
```go
warnings = maybeAppendWarnings(warnings, requestResources)
```

Reply:
good suggestion, I'll change it to something like that